tests/mongo.yml

```diff
@@ -0,0 +1,183 @@
+name: MongoDB Test Suite
+vars:
+  mongo_uri: mongodb://localhost:27017
+  mongo_database: venom
+  mongo_collection: cards
+
+testcases:
+  - name: Reset collection
+    steps:
+      - type: mongo
+        uri: "{{.mongo_uri}}"
+        database: "{{.mongo_database}}"
+        collection: "{{.mongo_collection}}"
+        actions:
+          - type: dropCollection
+          - type: createCollection
+
+  - name: Insert some data
+    steps:
+      - type: mongo
+        uri: "{{.mongo_uri}}"
+        database: "{{.mongo_database}}"
+        collection: "{{.mongo_collection}}"
+        actions:
+          - type: insert
+            documents:
+              - |
+                {
+                  "_id": {
+                    "$oid": "636a7c57d044a5a54b0f8cf7"
+                  },
+                  "color": "black",
+                  "value": "joker"
+                }
+              - |
+                {
+                  "color": "red",
+                  "value": "joker"
+                }
+
+          - type: insert
+            file: mongo/cards.json
+        info: "{{.result.actions}}"
+        assertions:
+          - result.actions.actions0.insertedids.__len__ ShouldEqual 2
```

Comment:
could you use assertion `ShouldHaveLength` instead of `__len__`, we will try to deprecate those old ugly properties in a near future